service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/RepresentativesInbox.java

```diff
@@ -31,17 +39,75 @@
             .filter(StringUtils::isNotBlank)
             .collect(Collectors.toCollection(LinkedHashSet::new));
 
-        emails.addAll(
-            nullSafeList(caseData.getRespondents1()).stream()
-                .filter(respondent ->
-                    (preference == RepresentativeServingPreferences.DIGITAL_SERVICE)
-                        == respondent.getValue().hasRegisteredOrganisation())
-                .map(respondent -> Optional.ofNullable(respondent.getValue().getSolicitor())
-                    .map(RespondentSolicitor::getEmail).orElse(null))
-                .filter(StringUtils::isNotBlank)
-                .collect(Collectors.toCollection(LinkedHashSet::new))
-        );
+        LinkedHashSet<String> respondentEmails = getRespondentSolicitorEmails(caseData.getRespondents1(),
```

Comment:
need to be linked?

Reply:
Previous code, leaving as is as discussed